app/backend/approaches/chatreadretrievereadvision.py

```diff
@@ -110,7 +110,7 @@ async def run_until_final_call(
         chat_completion: ChatCompletion = await self.openai_client.chat.completions.create(
             model=self.gpt4v_deployment if self.gpt4v_deployment else self.gpt4v_model,
             messages=messages,
-            temperature=overrides.get("temperature") or 0.0,
```

Comment:
This was a bug, temperature parameter shouldn't be used in the first call.

Reply:
Is there a doc that indicates this? Would be helpful to add it in a comment for folks reading this

Reply:
Sure, added comment to both chat files.